functional_tests/docker-compose-dev-app.yml

```diff
@@ -0,0 +1,38 @@
+services:
+  web:
+    extends:
+      file: ../docker-compose.yml
+      service: web
+    build:
+      context: ..
+      target: dev
+    environment:
+      DJANGO_SETTINGS_MODULE: cms.settings.functional_test
+    ports: !override
+      - 18000:8000 # runserver
+    volumes:
+      # Mount the repo at /app/
+      - ../:/app/
+    command: >
+      sh -c "poetry install &&
+             django-admin collectstatic --verbosity 0 --noinput --clear &&
```

Comment:
Are these two needed?. These are done when the docker image is built?.

Reply:
They probably shouldn't be needed, but I had issues with dependencies and statics files not being properly initialised locally and running them here was the only way to resolve. I'll check again, and if it's still an issue I'll add a comment explaining.